voter-dapp/src/ActiveRequests.js

```diff
@@ -21,6 +21,12 @@ function ActiveRequests() {
   })).account;
   const initialFetchComplete = pendingRequests && currentRoundId && votePhase && account;
 
+  const revealEvents = useCacheEvents(
+    "Voting",
+    "VoteRevealed",
+    useMemo(() => ({ filter: { voter: account, roundId: currentRoundId }, fromBlock: 0 }), [account, currentRoundId])
```

Comment:
(non actionable :) ) Maybe this is an overoptimization, but sometime down the line, we could configure the block where we deployed the system in a global config file and set that as the `fromBlock` so infura doesn't have to work as hard and we get a slightly faster result.

Reply:
That's a great idea, we could/should totally do that.